src/lexedata/importer/interleaved.py

```diff
@@ -76,12 +76,28 @@
             if len(cogsets) == 1 or len(cogsets) == len(forms):
                 True
             else:
-                print(
+                logger.warning(
                     "{:}: Forms ({:}) did not match cognates ({:})".format(
                         entry.coordinate, ", ".join(forms), ", ".join(cogsets)
                     ),
                     file=sys.stderr,
                 )
+            for form, cogset in zip(forms, cogsets + [None]):
+                w.writerow([language_name, concepts[c], form, None, cogset])
+
+
+if __name__ == "__main__":
+    import argparse
 
-            for form, comment, cogset in zip(forms, comments, cogsets + [None]):
-                w.writerow([language_name, concepts[c], form, comment, cogset])
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "excel", type=openpyxl.load_workbook, help="The Excel file to parse"
+    )
+    parser.add_argument(
+        "--directory",
+        type=Path,
+        default=Path(__file__).parent,
+        help="Path to directory where forms.csv is created (default: root directory of this script)",
```

Comment:
I would set the default to the current working directory. The script might be installed in the weirdest place in a python environment.